core/chains/evm/client/rpc_client.go

```diff
@@ -327,7 +327,7 @@ func (r *rpcClient) BatchCallContext(ctx context.Context, b []rpc.BatchElem) err
 	defer cancel()
 	lggr := r.newRqLggr().With("nBatchElems", len(b), "batchElems", b)
 
-	lggr.Trace("RPC call: evmclient.Client#BatchCallContext")
+	lggr.Debug("RPC call: evmclient.Client#BatchCallContext")
```

Comment:
I added that and some other temporary debug statements to help understand why the smoke tests are failing.  Will remove before merging, but took it out of draft because I'm stuck on the smoke test issues and want people to start reviewing it while I'm working with QA on figuring that issue out.